src/patternfly/layouts/Toolbar/styles.scss

```diff
@@ -2,16 +2,23 @@
 
 // The value of component scoped variables is always defined by a global variable:
 .pf-l-toolbar {
+  padding: 0 0 2em 2em;
```

Comment:
The toolbar shouldn't have padding (it isn't in the existing toolbar layout, and wasn't asked to be added as part of #740). Also (just a CSS syntax thing) you would use `padding-bottom` and `padding-left` instead of the shorthand (shorthand should only be used if it's required to set each value, and `padding-top` and `padding-right` here are unnecessary), and you would use css vars instead of `2em`.

Reply:
Thanks for the advice. The line of code has been removed from the latest build.